webapp/TargetedMS/js/QCTrendPlotPanel.js

```diff
@@ -184,10 +184,21 @@ Ext4.define('LABKEY.targetedms.QCTrendPlotPanel', {
             this.endDate = this.formatDate(this.calculateEndDateByOffset());
         }
 
-        // initialize the form panel toolbars and display the plot
-        this.add(this.initPlotFormToolbars());
+        this.getExpRunRangeDetails();
+    },
 
-        this.displayTrendPlot();
+    getExpRunRangeDetails: function() {
+        var urlParams = LABKEY.ActionURL.getParameters();
+        this.showExpRunRange = urlParams['RunId'] !== undefined && urlParams['RunId'] !== "null" && urlParams['RunId'] !== "";
```

Comment:
Can we avoid the "null" string here too? What are you guarding against? If you're concerned about bogus values on the URL, I assume non-numerics would trip up the code too.